Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:14.15.1 AS build
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+ARG sockets_address=http://localhost:8000
+ENV REACT_APP_SOCKETS_ADDRESS=$sockets_address
+ARG react_app_server=http://localhost:8008
```

Comment:
the same comment applied to those!

Reply:
I will change it to app_server and app_frontend